.circleci/config.yml

```diff
@@ -111,17 +130,36 @@ jobs:
             # highlighted in development.
             ./script/linting/lint --extend-ignore=T000
 
+  build:
+    <<: *parametrised-python-executor
+
+    steps:
+      - checkout
+
+      - *build-checksum-file
+      - *restore-dependencies-cache
+      - *install-dependencies
+      - *save-dependencies-cache
+
+      - run:
+          name: Build only
+          command: |
+            source venv/bin/activate
+            ./script/release/build.sh
+
   release:
     docker:
       - image: cimg/python:3.9
     steps:
       - checkout
 
+      - *build-checksum-file
+      - *install-dependencies
+
       - run:
-          name: Setup virtualenv
+          name: Install release requirements
           command: |
-            python3 -m venv venv
-            venv/bin/pip install -r script/release/requirements.txt
+            venv/bin/pip install -r script/release/requirements.txt -r all-requirements.txt
```

Comment:
I'm now questioning whether we've actually done a release with this CI before, since I think the docs build would have always needed to have everything else installed. Anyway, now we have this repeated outside the release step I'm pretty confident it'll work.